modules/apps/data-engine/data-engine-rest-test/src/testIntegration/java/com/liferay/data/engine/rest/resource/v2_0/test/DataRecordResourceTest.java

```diff
@@ -469,6 +471,52 @@ public void testPostDataRecordCollectionDataRecord() throws Exception {
 			404,
 			dataRecordResource.postDataRecordCollectionDataRecordHttpResponse(
 				RandomTestUtil.randomLong(), randomDataRecord()));
+
+		// --> Persist data for text in a fieldset
+
+		DataDefinition postedDataDefinition =
```

Comment:
rename `postedDataDefinition` to dataDefinition

Reply:
Okie dokie :+1: